File: Chapter 14/Address Sanitizer/Address Sanitizer/ViewController.m
//
//  ViewController.m
//  Address Sanitizer
//
//  Created by Jak Tiano on 10/24/16.
//  Copyright © 2016 PacktPub. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    int* myArray = malloc( sizeof(int) * 2 );
    myArray[0] = 0;
    myArray[1] = 1;
    
    NSLog(@"%i", myArray[3]);
}



- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Chapter 2/Resources/HelloWorld/HelloWorld/ViewController.swift
//
//  ViewController.swift
//  HelloWorld
//
//  Created by Jak Tiano on 7/14/16.
//  Copyright © 2016 PacktPub. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var textLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        textLabel.text = ""
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func buttonWasPressed(_ sender: AnyObject) {
        if textLabel.text == "" {
            textLabel.text = "Hello World!"
        } else {
            textLabel.text = ""
        }
    }
}
